service/src/main/java/bio/terra/workspace/app/controller/ControlledAzureResourceApiController.java

```diff
@@ -126,6 +129,35 @@ public ControlledAzureResourceApiController(
     return new ResponseEntity<>(response, HttpStatus.OK);
   }
 
+  @Override
+  public ResponseEntity<ApiCreateControlledAzureRelayNamespace> createAzureRelayNamespace(
+      UUID workspaceId, @Valid ApiCreateControlledAzureRelayNamespaceRequestBody body) {
+    features.azureEnabledCheck();
+
+    final AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
+    ControlledResourceFields commonFields =
+        toCommonFields(workspaceId, body.getCommon(), userRequest);
+
+    ControlledAzureRelayNamespaceResource resource =
+        ControlledAzureRelayNamespaceResource.builder()
+            .common(commonFields)
+            .namespaceName(body.getAzureRelayNamespace().getNamespaceName())
+            .region(body.getAzureRelayNamespace().getRegion())
+            .build();
+
+    final ControlledAzureRelayNamespaceResource created =
+        controlledResourceService
+            .createControlledResourceSync(
```

Comment:
Out of scope for this PR: It's unclear to me when we do things synchronously vs asynchronously in WSM. Creating controlled resources like this seems like it'd be a potentially long-running operation that merits dispatching asynchronously and returning a job ID to the client for polling.

Reply:
Here is the relevant document [MC Terra - Async REST API Specification](https://docs.google.com/document/d/1PTd4xvmV9xnEkWaIgFc6d3VUyJwwymQTFdhpKrdsUKw/edit#heading=h.ol9mx3vfhjjj)